internal/pkg/views/list.go

```diff
@@ -254,31 +254,30 @@ func (w *ListWidget) Navigate(nodes []*expanders.TreeNode, content *expanders.Ex
 		eventing.Publish("list.navigated", ListNavigatedEventState{Success: false})
 	}
 	currentItem := w.CurrentItem()
+
+	parentNodeID := "root"
+	nodeID := "root"
+	if w.expandedNodeItem != nil {
+		parentNodeID = w.expandedNodeItem.ID
+		nodeID = currentItem.ID
+	}
+
+	w.contentView.SetContent(currentItem, content.Response, content.ResponseType, title)
 	if len(nodes) > 0 {
 		w.expandedNodeItem = currentItem
 		w.SetNodes(nodes)
-	}
 
-	w.contentView.SetContent(content.Response, content.ResponseType, title)
-	if currentItem != nil {
-		w.title = w.title + ">" + currentItem.Name
+		if currentItem != nil {
+			w.title = w.title + "fooo>" + currentItem.Name
```

Comment:
```suggestion
			w.title = w.title + ">" + currentItem.Name
```

Reply:
![image](https://user-images.githubusercontent.com/1939288/76203542-b9784880-61ee-11ea-97fc-ffbce7467579.png)

Nit: guess this was for testing

Reply:
Fixed